sdk/rust/oak_tests/src/lib.rs

```diff
@@ -408,6 +428,53 @@ impl OakRuntime {
         let half = self.grpc_in_half.as_ref()?;
         Some(half.channel.clone())
     }
+    fn node_create(
+        &mut self,
+        node_name: String,
+        contents: &str,
+        handle: oak::Handle,
+    ) -> Result<(NodeMain, String, oak::Handle), OakStatus> {
```

Comment:
I suggest creating a struct to hold this return value at some point.

Reply:
Done.